src/main/java/views/add_edit_epense_views/AddExpenseV.java

```diff
@@ -24,14 +23,15 @@ public class AddExpenseV extends Component implements ActionListener, LoadMonthM
     private final JComboBox<String> categoryCombo;
     private final String selectedExpense;
     private String selectedCategory;
-    private JCheckBox isRecurringCheckBox;
+    private final JCheckBox isRecurringCheckBox = new JCheckBox("This is a recurring expense.");
```

Comment:
I was testing the view, and I think having a whole sentence commenting the check box may be too long. Make it shorter, like "Is recurring expense".